services/minespace-web/src/components/pages/Project/ProjectPage.js

```diff
@@ -97,6 +99,11 @@ export class ProjectPage extends Component {
                 <Tabs.TabPane tab="Overview" key="overview">
                   <ProjectOverviewTab />
                 </Tabs.TabPane>
+                {!IN_PROD() && (
```

Comment:
Just available in PROD

Reply:
To confirm, this means available everywhere except Prod :)